controllers/user.js

```diff
@@ -54,28 +82,27 @@ async function verify(req, res) {
     req.app.locals.breaches
   );
 
-  const supportedLocales = EmailUtils.getSupportedLocales(req);
-  const buttonValue = req.fluentFormat("report-scan-another-email");
-  const whichView = "email_partials/report";
-
   await EmailUtils.sendEmail(
     verifiedEmailHash.email,
     req.fluentFormat("user-verify-email-report-subject"),
     "default_email",
     {
-      supportedLocales,
+      supportedLocales: EmailUtils.getSupportedLocales(req),
       email: verifiedEmailHash.email,
-      date: HBSHelpers.prettyDate(supportedLocales, new Date()),
+      date: HBSHelpers.prettyDate(req.supportedLocales, new Date()),
```

Comment:
Is `EmailUtils.getSupportedLocales(req)` (L90) different than the `req.supportedLocales` (L92)?

It looks like `req.supportedLocales` gets set in the `pickLanguage` middleware via

https://github.com/mozilla/blurts-server/blob/61d2a44d7ab55abd3325092276b8681143e71061/middleware.js#L27-L49

Reply:
I think there was a subtle difference, but I've forgotten what it was. It may just be dupe logic that can be removed.

Reply:
Dupe. Except for `res.vary`. Could the lack of `res.vary` in `EmailUtils.getSupportedLocales` have caused #630 ? Will get rid of it.

Reply:
Ooo, that's a good insight and idea. It's worth investigating for sure.